WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -168,6 +225,18 @@ private extension HelpAndSupportViewController {
         )
     }
 
+    /// Contact WCPay Support cell.
+    ///
+    func configureContactWCPaySupport(cell: ValueOneTableViewCell) {
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+        cell.textLabel?.text = NSLocalizedString("Contact WCPay Support", comment: "Contact WCPay Support title")
```

Comment:
Should probably say "WooCommerce Payments" and not "WCPay" - "WCPay" is a internal term that won't be familiar to/recognizable by merchants